src/main/java/io/getunleash/Segment.java

```diff
@@ -0,0 +1,72 @@
+package io.getunleash;
+
+import io.getunleash.lang.Nullable;
+
+import java.util.List;
+
+public class Segment {
+    private int id;
+    private String name;
+    @Nullable
+    private String description;
+    private List<Constraint> constraints;
+    private String createdBy;
+    private String createdAt;
```

Comment:
I'm not sure we need description, createdBy and createdAt, they may be part of the body, but it's not really needed for performing the constraint matching, maybe drop down to just `id, name, constraints`?

I think we've already configured Jackson to parse without failing on unknown fields, so it should be legal

Reply:
Removed